tests/test_forgejo/results/push.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2024-2025 Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+"""
+Result specification for Forgejo "push" event
+"""
+
+summary = "testuser-w2fm created push on fedora/w2fm-test"
+
+specification = """Event: Push
+Repository: fedora/w2fm-test (https://codeberg.org/fedora/w2fm-test)
+Pusher: Akashdeep Dhar (https://codeberg.org/gridhead)
+Branch: pull
+Commits:
+Commit: Taking a dump with `json.dumps()` function
+
+Signed-off-by: Akashdeep Dhar <testuser-w2fm@fedoraproject.org>
+ by Akashdeep Dhar (https://codeberg.org/fedora/w2fm-test/commit/1c5ccba31592d36a7b999434768c7498edfec822)"""
```

Comment:
It would be preferrable to keep only the first line of the commit message. (see below as well)